lib/oli_web/live/projects/visibility.ex

```diff
@@ -28,110 +28,105 @@ defmodule OliWeb.Projects.VisibilityLive do
 
   def render(assigns) do
     ~L"""
-    <div class="card">
-      <div class="card-body ">
-        <div class="row ">
-          <div class="col-sm-8 p-3">
-            <h4 class="card-title">Course builder visibility</h4>
-          </div>
-        </div>
-        <div class="row">
-          <div class="col-sm-8">
-            <div class="form-check form-switch">
-              <form phx-change="option" id="visibility_option">
-                <div class="form-group">
-                  <%= radio_button :visibility, :option, "authors", class: "form-check-input", checked: if  @project.visibility == :authors, do: true, else: false  %>
-                  <%= label :visibility_option, "Only direct authors to this course", class: "form-check-label" %>
-                </div>
-                <div class="form-group">
-                  <%= radio_button :visibility, :option, "global", class: "form-check-input", checked: if  @project.visibility == :global, do: true, else: false %>
-                  <%= label :visibility_option, "Any user with instructor access", class: "form-check-label" %>
-                </div>
-                <div class="form-group">
-                  <%= radio_button :visibility, :option, "selected", class: "form-check-input", checked: if  @project.visibility == :selected, do: true, else: false %>
-                  <%= label :visibility_option, "Only the list of selected users or institutions below", class: "form-check-label" %>
+    <div class="row py-5 border-bottom">
+      <div class="col-md-4">
+        <h4>Publishing Visibility</h4>
+        <div class="text-muted">Control who can create course sections for this project once it is published.</div>
+      </div>
+      <div class="col-md-8">
+        <form phx-change="option" id="visibility_option">
+          <div class="form-check form-switch">
+            <div class="form-group">
+              <%= label class: "form-check-label" do %>
+                <%= radio_button :visibility, :option, "authors", class: "form-check-input", checked: @project.visibility == :authors or is_nil(@project.visibility) %>
+                <div class="d-flex align-items-center">
+                  <p>Project authors</p>
+                  <span class="ml-2 badge badge-pill badge-light">default</span>
                 </div>
-              </form>
+                <small>Only authors added as collaborators to this project</small>
```

Comment:
Consider rephrasing: "Only instructors who are collaborators and have linked their accounts". We want to try and minimize confusion by using the correct domain roles. From the user delivery domain, it's really instructors who have access